### PR TITLE
Don't collapse one line of code

### LINKED ISSUE #556: Don’t show the expand button for one line of code
[I’m not sure if this is the right repo.]

Here’s an example:
Screen Shot 2019-06-30 at 7 25 42 PM

It seems like we should not collapse the code when only one line needs to be collapsed. It should look like this:
Screen Shot 2019-06-30 at 7 25 48 PM

### PR DESCRIPTION
**Pull request by cherifGsoul**
> This is needed by https://github.com/canjs/bit-docs-html-canjs/issues/556

This prevent collapsing one line of code, for example the following will not collapse line 1 and 8:

```html
// Creates a mock backend with 3 todos
import { todoFixture } from "//unpkg.com/can-demo-models@5";
todoFixture(3);

import { Component } from "//unpkg.com/can@5/core.mjs";

Component.extend({
	tag: "todos-app",
	view: `
		<h1>Today’s to-dos</h1>
	`,
	ViewModel: {
	}
});
<span line-highlight='2-7,9-14,only'/>
```
Fixes https://github.com/canjs/bit-docs-html-canjs/issues/556

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -66,11 +66,14 @@ var getConfig = function(lineString, lineCount) {
 					return typeof val === 'number' && !isNaN(val);
 				})
 			;
-
+			
 			if (range[0] > current + padding) {
-				collapse.push(current + '-' + (range[0] - 1 - padding));
+				var collapseEnd = (range[0] - 1 - padding);
+				if (collapseEnd !== current) {
```
Comment (justinbmeyer) [inline index.js:72]: @cherifGsoul can you explain this logic change? Would this show the lines between a set of highlights like:

```
@highlight 1-3,33-55
```

`range[0] > current + padding` would be true when `range[0]` is `33`

Comment (cherifGsoul) [reply index.js:72]: @justinbmeyer The logic is to compare the first line and the last line of collapsible:
- Using `current` which is initialized to `1` or calculated in this expression at the end of the loop `current = (range[1] || range[0]) + padding + 1`
- In each iteration of the loop `collapseEnd` is calculated based on the first item `range[0]` of the new range of line highlight `collapseEnd = (range[0] - 1 - padding)`

In short:
`current = currentRange[1] + 1 + padding`
`next = nextRange[0] - 1 - padding`

`next - current` should be greater than `1` to make a collapsible

So for `@highlight 1-3,33-55` we have a collapsible between `[7, 29]` and it shows only the 3 lines for padding between highlights:
`current = 3 + 1 + 3 // => start collapsible from line 7`
`next = 33 - 1 - 3 // => close collapsing at the line 29`

**Review by justinbmeyer — APPROVED** [review]
